src/main/java/frc/robot/subsystems/auto/AutoLogic.java

```diff
@@ -278,7 +280,7 @@ public static Command scoreCommand() {
   public static Command intakeCommand() {
     if (r.superStructure != null) {
       if (ARMSENSOR_ENABLED) {
-        return Commands.sequence(r.superStructure.coralPreIntake(), r.superStructure.coralIntake())
+        return Commands.sequence(r.superStructure.coralIntake())
```

Comment:
```suggestion
        return r.superStructure.coralIntake()
```
A sequence containing one command does the same thing as running that one command.